packages/core/crates/core-packet/src/interaction.rs

```diff
@@ -1015,11 +1015,8 @@ impl PacketInteraction {
             }});
 
         spawn_local(async move {
-            processing_stream
-                .map(|x| Ok(x))
-                .forward(futures::sink::drain())
-                .await
-                .unwrap();
+            // poll the stream until it's done
+            while processing_stream.next().await.is_some() {}
```

Comment:
How is this different?

Reply:
It isn't. Only difference is that this does not require `futures`. Only requires `futures-lite`